modules/minimp3/audio_stream_mp3.cpp

```diff
@@ -143,6 +145,25 @@ void AudioStreamPlaybackMP3::tag_used_streams() {
 	mp3_stream->tag_used(get_playback_position());
 }
 
+void AudioStreamPlaybackMP3::set_parameter(const StringName &p_name, const Variant &p_value) {
+	if (p_name == SNAME("looping")) {
+		if (p_value == Variant()) {
+			looping_override = false;
+			looping = false;
+		} else {
+			looping_override = true;
+			looping = p_value;
+		}
+	}
+}
+
+Variant AudioStreamPlaybackMP3::get_parameter(const StringName &p_name) const {
+	if (looping_override && p_name == SNAME("looping")) {
+		return looping;
+	}
+	return Variant();
+}
+
```

Comment:
In line 191, add this
```c++
mp3s->looping = loop;
```
to inherit looping setting from stream and be in sync with ogg vorbis.

Reply:
ah that line should actually be removed instead, since its ignored.

Reply:
Ah, I get it. If the overwrite is deactivated, the inspector can't read `looping` and the audio player mustn't read it.